manager/assets/modext/widgets/security/modx.panel.groups.roles.js

```diff
@@ -146,11 +162,14 @@ Ext.extend(MODx.panel.GroupsRoles,MODx.FormPanel,{
         var usergroup = id.replace('n_ug_', '') - 0; // typecasting
         
         var userGrid = Ext.getCmp('modx-usergroup-users');
+        var westPanel = Ext.getCmp('modx-tree-usergroup').ownerCt.ownerCt.layout.west;
```

Comment:
I guess `Ext.getCmp('modx-leftbar-tabs`) might be "safer" in case the structure changes